spec/decrypter_spec.rb

```diff
@@ -0,0 +1,36 @@
+require './lib/decrypter'
+
+RSpec.describe Decrypter do
+  let(:decrypter) { Decrypter.new(
+    "keder ohulw", 
+    "02715", 
+    "040895"
+  )}
```

Comment:
When declaring a multi-line `let`, it's preferred to wrap everything in an explicit block:

```
  let(:decrypter) do
    Decrypter.new(
      "keder ohulw", 
      "02715", 
      "040895"
    )
  end
```